packages/components/tree-v2/src/tree-node.vue

```diff
@@ -19,7 +19,10 @@
   >
     <div
       :class="ns.be('node', 'content')"
-      :style="{ paddingLeft: `${(node.level - 1) * indent}px` }"
+      :style="{
+        paddingLeft: `${(node.level - 1) * indent}px`,
+        height: itemSize + 'px',
+      }"
```

Comment:
**Suggestion:** we do not really need this since we have style being passed into this component, and it contains the height we needed here, see this screenshot 👇 

![CleanShot 2023-02-12 at 00 14 40@2x](https://user-images.githubusercontent.com/15975785/218268757-e83f1277-8012-4a2d-80fa-ae26b1db60ad.png)

![CleanShot 2023-02-12 at 00 15 57@2x](https://user-images.githubusercontent.com/15975785/218268832-3992c853-8e25-481b-ab53-10e44e822d5d.png)

Reply:
It should still be necessary here, the code in the figure will set the height of `el-tree-node`, but the height of `el-tree-node__content` is set to `26px`.
https://github.com/element-plus/element-plus/blob/06126ea57123ed6cba058a7465123e04865d396f/packages/theme-chalk/src/tree.scss#L63

Reply:
Maybe, `height: inherit` here is more appropriate?

Reply:
<img width="1240" alt="image" src="https://user-images.githubusercontent.com/27342882/218289738-f4e165a6-58d7-407b-ab4b-7e53286adeef.png">

Since the height is already set in the parent, there seems to be no problem if you set the height to inherit(`height: inherit`) as @jw-foss said!

Reply:
`tree` and `tree-v2` use the same style file, such modification may affect `tree`.

Reply:
@ryuhangyeong Yes, as shown in your screenshot, `el-tree-node` has set the height, but it is not set in the `el-tree` component. Therefore, I cannot directly modify the `tree.scss` file.  If setting inline styles like I do now, I think it might be more intuitive to use `itemSize`?

Reply:
@btea oh my mistake. I agree with you!